test/date_extractor.js

```diff
@@ -0,0 +1,73 @@
+import extractor from "../src/extractor";
+import cheerio from "cheerio";
+const fs = require("fs");
+const spawn = require("child_process").spawn;
+import pythonBridge from 'python-bridge';
+
+import PythonAdapter from './bridge';
+const python = new PythonAdapter()
+
+const HTML = require("./withHTML.json");
+const { promisify } = require("util");
+const writeFile = promisify(fs.appendFile);
+const OUTPUT_FNAME = "./test/withDate.json";
+
+const getPythonDate = (url) =>
```

Comment:
can we do this test with a Python adapter instead of spawn?

Reply:
this code was only for manual testing.
removed